packages/assets-controllers/src/TokensController.ts

```diff
@@ -74,7 +70,7 @@ export type SuggestedAssetMetaBase = {
   time: number;
   type: string;
   asset: Token;
-  interactingAddress?: string;
+  interactingAddress: string;
```

Comment:
The only place where this object is created is in the method `watchAsset` and it uses:

```
interactingAddress: interactingAddress || selectedAddress
```

The value of `selectedAddress` is defined as a `string`, not `string | undefined`, so this value is never undefined.